share/instruments/instruments.xml

```diff
@@ -5124,7 +5124,7 @@
             <name>Percussion - Unpitched</name>
             <Instrument id="drumset">
                   <longName>Drumset</longName>
-                  <shortName>Drs.</shortName>
+                  <shortName>D. Set</shortName>
```

Comment:
Conflicts with mtest for importmidi, see https://travis-ci.org/musescore/MuseScore/jobs/363144548#L5031-L5065 and more